scootapi/server/setup.go

```diff
@@ -32,7 +34,9 @@ func makeServers(
 func Defaults() (*ice.MagicBag, jsonconfig.Schema) {
 	bag := ice.NewMagicBag()
 	bag.PutMany(
-		endpoints.MakeStatsReceiver,
+		func(scope endpoints.StatScope) stats.StatsReceiver {
```

Comment:
Right - this is for how the internal stats receiver is handing time/numeric values.  We are currently reporting stuff like "X took 1000000 ms" when it really took 1ms.